dbt/include/redshift/macros/relations/materialized_view/describe.sql

```diff
@@ -24,6 +24,22 @@
     {%- endset %}
     {% set _materialized_view = run_query(_materialized_view_sql) %}
 
+    {%- set _column_descriptor_sql -%}
+        SELECT
+            n.nspname AS schema,
```

Comment:
Per the comment above, we can drop `schema` and `table` here, and we should keep the sort key order if it is non-zero.

Reply:
fixed